src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/NuGet/ProjectAssetFileWatcher.cs

```diff
@@ -81,8 +87,10 @@ protected override void Initialize()
         /// <summary>
         /// Called on changes to the project tree.
         /// </summary>
-        internal void DataFlow_Changed(IProjectVersionedValue<Tuple<IProjectTreeSnapshot, IProjectSubscriptionUpdate>> dataFlowUpdate)
+        internal async Task DataFlow_Changed(IProjectVersionedValue<Tuple<IProjectTreeSnapshot, IProjectSubscriptionUpdate>> dataFlowUpdate)
         {
+            await _task.ConfigureAwait(false);
```

Comment:
Wasn't sure whether to make this change or not.  Conceptually, if this method is being invoked, then `InitializeCoreAsync` should be done already.  Thoughts?

Reply:
You can simply await it.